test/test_transforms_v2_functional.py

```diff
@@ -462,9 +444,12 @@ def test_dispatcher_kernel_signatures_consistency(self, dispatcher_info, datapoi
         kernel_params = list(kernel_signature.parameters.values())[1:]
 
         # We filter out metadata that is implicitly passed to the dispatcher through the input datapoint, but has to be
-        # explicit passed to the kernel.
-        datapoint_type_metadata = datapoint_type.__annotations__.keys()
```

Comment:
We need to be explicit now, since except for `datapoints.BoundingBoxes` the other datapoints no longer have an `__annotations__` parameter. I was confused at first, but then I remembered

https://github.com/pytorch/vision/blob/16d62e3072955bd92b76a4ae73fefa73ecc9ee3e/torchvision/datapoints/_datapoint.py#L19

But that was removed with the commit.